lua/plugins/init.lua

```diff
@@ -314,6 +314,11 @@ return packer.startup(function()
       disable = not plugin_status.vim_fugitive,
       cmd = {
          "Git",
+         "Gdiff",
```

Comment:
Gdiff is short for Gdiffsplit. I only added Gdiffsplit here and Gdiff works for me.